### PR TITLE
2638 create a course minor changes

### PR DESCRIPTION
**Pull request by defong**
### Context
Content updates

### Changes proposed in this pull request
Fixed page title consistence to be the same as the h1.
Amend content for GCSE requirements for applicants 

### Guidance to review

Create course 
browser tab show be the same as h1

What type of course? 
image

Pick a primary subject 
image

Pick a secondary subject 
image

Specify an age range
image

Pick a course outcome
image

Is it fee paying or salaried?

image

Full time or part time?

image

Pick the locations for this course
image

Who is the accredited body? 

image

GCSE requirements for applicants 
image

When will applications open? 

image

When does the course start?
image

Check your answers before confirming 
image

Is this a teaching apprenticeship?

image

Content change for GCSE requirements for applicants page content

image

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: app/views/courses/_new_fields_holder.html.erb [hunk 1]
```diff
@@ -13,8 +13,8 @@
   <%= yield fields %>
 <% end %>
 
-<p class="govuk-body">
+<p class="govuk-body ">
```
Comment (dankmitchell) [inline app/views/courses/_new_fields_holder.html.erb]: Extra space here...

File: app/views/courses/accredited_body/edit.html.erb [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<%= content_for :page_title, "Pick an accredited body  – #{course.name_and_code}" %>
+<%= content_for :page_title, "Who is the accredited body?  – #{course.name_and_code}" %>
```
Comment (dankmitchell) [inline app/views/courses/accredited_body/edit.html.erb]: Extra space after `?`  ?

File: app/views/courses/accredited_body/new.html.erb [hunk 3]
```diff
@@ -1,15 +1,17 @@
...
+<h1 class="govuk-heading-xl">
+  Who is the accredited body?
+</h1>
+
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
-    <h1 class="govuk-heading-xl">
-      Who is the accredited body?
-    </h1>
+
```
Comment (dankmitchell) [inline app/views/courses/accredited_body/new.html.erb]: Extra space here

File: app/views/courses/entry_requirements/new.html.erb [hunk 4]
```diff
@@ -13,15 +13,16 @@
...
     <p class="govuk-body">
-      By law, all applicants need GCSE grade C/grade 4, or an equivalent, in English and Mathematics – plus Science for Primary courses.
-    </p>
-
-    <p class="govuk-body">
-      Candidates may still be taking a GCSE when they apply. Alternatively, they can take an equivalency test to demonstrate they have the required skills to start a course.
+      By law, all applicants need GCSE grade C/grade 4, or an equivalent, in English and maths - plus science for primary courses.
     </p>
 
+    <p class="govuk-body">However, they may still apply if: </p>
```
Comment (dankmitchell) [inline app/views/courses/entry_requirements/new.html.erb]: Extra space here...

**Comment by defong** [comment]
[Create a course minor changes](https://trello.com/c/AbWsHQsl/2638-create-a-course-minor-changes)

**Pull request merged**